devtools/materialize_asset.py

```diff
@@ -24,28 +24,26 @@ def main(asset_id):
     Then creates a job with asset selection.
     """
     # TODO (daz/zach): maybe there's a way to do this directly with dagster cli?
-    defs = Definitions(
-        assets=etl.default_assets,
-        resources=etl.default_resources,
-        jobs=[
-            define_asset_job(
-                name="materialize_one",
-                selection=AssetSelection.keys(asset_id).upstream(),
-                config={
-                    "resources": {
-                        "dataset_settings": {
-                            "config": EtlSettings.from_yaml(
-                                importlib.resources.path(
-                                    "pudl.package_data.settings", "etl_fast.yml"
-                                )
-                            ).datasets.dict()
+    pkg_source = files("pudl.package_data.settings").joinpath("etl_fast.yml")
```

Comment:
Neat! Wonder why they deprecated `importlib.resources.path` though 🤷

Reply:
It definitely was not clear to me from the docs. But `importlib.resources` has always seemed mysterious to me.